cpp/ps0007/cpp_ps0007_solution.cpp

```diff
@@ -25,9 +29,13 @@ int main()
 		cin >> dimention[0] >> dimention[1];
 	}
 	else
+	{
 		cout << "the program can't be calculated\n";
+		return 1;
+	}
 
-	cout << "the shape is:\n" << Area(shape, dimention);
+	// printing out the area of the shape 1/1
+	cout << "the area of the shape is:\n" << Area(shape, dimention);

```

Comment:
- [x] invoked function to calculate the area of the shape
- [x] used `cout::std` in a right way to print out the area